packages/docs/scripts/template.hbs

```diff
@@ -0,0 +1,6 @@
+{{#module}}
+{{>body~}}
+{{>member-index~}}
+{{>separator~}}
+{{>members~}}
+{{/module}}
```

Comment:
```suggestion
{{/module}}

```

Reply:
Actually, this is old stuff I am not using. Removing it.